modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2TeamMembers.path

```diff
@@ -21,28 +21,33 @@
 	<td></td>
 </tr>
 <tr>
-	<td>DROPDOWN</td>
-	<td>//select[contains(@class, 'form-control')]</td>
+	<td>CLOSE_BUTTON</td>
+	<td>//button[contains(@class, 'navbar-breakpoint')]</td>
 	<td></td>
 </tr>
 <tr>
 	<td>EMAIL_FIELD</td>
 	<td>//input[contains(@name, 'invites[${key_fieldNumber}]')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ICON_CHECK</td>
+	<td>//td[contains(@class, 'table-column-text-center')]</td>
+	<td></td>
+</tr>
 <tr>
 	<td>INVITE_BUTTON</td>
 	<td>//button[contains(@class,'invite-button')]</td>
 	<td></td>
 </tr>
 <tr>
-	<td>ROLE_DISABLED</td>
-	<td>//select[contains(@name, 'invites[${key_index}]')]//option[contains(text(), '${role}')][@disabled]</td>
+	<td>ERROR_MESSAGE</td>
```

Comment:
@luanakarla wold it be better to create a generic ERROR_MESSAGE path that don't depends on the text content?

Reply:
Done